Source/FieldSolver/PicsarHybridSpectralSolver/PicsarHybridFFTData.H

```diff
@@ -0,0 +1,32 @@
+#ifndef WARPX_PICSAR_HYBRID_FFTDATA_H_
+#define WARPX_PICSAR_HYBRID_FFTDATA_H_
```

Comment:
The contents of this file has been moved from the `WarpX.H` file ; it contains the definition of the `struct` `FFTData`